server/plugin/command.go

```diff
@@ -279,6 +279,10 @@ func (p *Plugin) handleSubscriptionsList(_ *plugin.Context, args *model.CommandA
 }
 
 func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs, parameters []string, userInfo *GitHubUserInfo) string {
+	if len(parameters) == 0 {
+		return "Please specify a repository."
+	}
```

Comment:
Looks good, just one request after the latest change. Since we moved the above block into this function, we also need to apply it to the `handleUnsubscribe` function.

Otherwise this looks great :+1:

@DHaussermann @dipak-demansol I'd like to get this merged so we can include it in the upcoming release, as it fixes a potential crash issue.

Reply:
Oddly enough, that function already has it and works properly. I'm really confused about why that would be, because I don't remember adding it, and before I added the code in this PR, it errored as you saw. Anyways, it is there and it works properly.

Reply:
Great work, LGTM :tada: